### PR TITLE
Add uploading file from object

### PR DESCRIPTION
**Pull request by bautrukevich**

### CODE REVIEW

File: src/fileFrom/fileFrom.ts [hunk 1]
```diff
@@ -1,63 +1,105 @@
...
+    cancel,
+    onProgress,
+
+    source,
+    integration,
+
+    retryThrottledRequestMaxTimes,
+
+    baseCDN = defaultSettings.baseCDN,
+    defaultEffects
```
Comment (jeetiss) [inline src/fileFrom/fileFrom.ts]: we receive `defaultEffects` from fileInfo
see https://uploadcare.com/docs/api_reference/upload/groups/#example

Comment (bautrukevich) [reply src/fileFrom/fileFrom.ts]: Hmm, but in https://uploadcare.com/api-refs/upload-api/#operation/fileUploadInfo there is no any word about this. @andrewshkovskii , can you help?

Comment (jeetiss) [reply src/fileFrom/fileFrom.ts]: because this works only in groups
https://uploadcare.com/api-refs/upload-api/#operation/filesGroupInfo

**Comment by bautrukevich** [comment]
Resolved!

**Comment by bautrukevich** [comment]
Fixed

**Pull request merged**